vspace/src/main/webapp/WEB-INF/views/staff/files/file.html

```diff
@@ -3,8 +3,6 @@
 <link rel="stylesheet" type="text/css" id="applicationStylesheet"
     th:href="@{/resources/extra/Home.css}" />
 
-<script src="//use.fontawesome.com/releases/v5.8.1/js/all.js"
```

Comment:
is the js needed somewhere? if so, it needs to be added to the layout as well.

Reply:
added it in both the layout files - main.html and main_staff.html